server/src/test/java/io/crate/protocols/postgres/PostgresWireProtocolTest.java

```diff
@@ -272,9 +272,9 @@ public void testDescribePortalMessage() throws Exception {
             ByteBuf response = channel.readOutbound();
             try {
                 assertThat(response.readByte()).isEqualTo((byte) 'T');
-                assertThat(response.readInt()).isEqualTo(46);
+                assertThat(response.readInt()).isEqualTo(42);
                 assertThat(response.readShort()).isEqualTo((short) 1);
-                assertThat(PostgresWireProtocol.readCString(response)).isEqualTo("($1 = ANY([1, 2, 3]))");
+                assertThat(PostgresWireProtocol.readCString(response)).isEqualTo("($1 IN (1, 2, 3))");
```

Comment:
How are these changes related?

Reply:
The `IN` is converted to an `ANY()` function call here:https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/analyze/relations/select/SelectAnalyzer.java#L62

Previously we'd just the `toColumn` when sending the row description here: https://github.com/crate/crate/pull/17858/files#diff-c6c87b42c88c54675f8ca239e2da30e854c4503fd570aa6ac28dd10ea3cbf794L444

Reply:
Is your link really pointing to the right code? 
Anyway, thanks for the explanation.